aiida_mlip/data/model.py

```diff
@@ -35,7 +35,7 @@ class ModelData(SinglefileData):
         Set the file for the node.
     from_local(file, architecture, filename=None):
         Create a ModelData instance from a local file.
-    from_url(url, architecture, filename=None, cache_dir=None, keep_file=False)
+    from_uri(url, architecture, filename=None, cache_dir=None, keep_file=False)
```

Comment:
Are you sure you want `URI` here? `URL`s are a subset of `URI`s which provide a resource, but `URI`s also include `URN`s which have no way to reach them and only provide a base-name mapping (from which we couldn't get a file).

If we do want this, need to change all instances (including arguments) to `uri`.